frontend/angular.json

```diff
@@ -59,8 +60,8 @@
                                 },
                                 {
                                     "type": "anyComponentStyle",
-                                    "maximumWarning": "6kb",
-                                    "maximumError": "10kb"
+                                    "maximumWarning": "10kb",
+                                    "maximumError": "15kb"
```

Comment:
do not change angular.json

Reply:
We can't decrease size of schedule-definition.component.sass. Any extra imports or unused styles. For this reason budgets was changed